build.sbt

```diff
@@ -1,6 +1,4 @@
-scalaVersion := "2.10.6"
-
-crossScalaVersions := Seq("2.10.6", "2.11.8")
+scalaVersion in ThisBuild := "2.11.8"
```

Comment:
Samza doesn't have build for 2.12 yet, they merged PR with samza support but haven't released it yet (https://github.com/apache/samza/pull/82). We can add 2.12 support but build samza for 2.11 only (as we do now for 2.10).